relay-cardinality/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "relay-cardinality"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Metrics Cardiniality Limiter"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "23.10.1"
+edition = "2021"
+license-file = "../LICENSE"
+publish = false
+
+[features]
+redis = ["relay-redis/impl"]
+
+[dependencies]
+hash32 = "0.3.1"
+itertools = { workspace = true }
+relay-log = { path = "../relay-log" }
+relay-metrics = { path = "../relay-metrics" }
```

Comment:
I agree, the API needs more work, it is intentionally stupid because I am not sure what the total set of requirements will be. So my approach was to do the dumbest thing first than gradually improve on it.

E.g. just accepting hashes isn't enough, it would need to include some kind of scope as well and we're not even sure yet what scopes we want to support or wether even the scope should be generic.

Reply:
Addressed with a new trait now `CardinalityItem`